### DISCUSSION TITLE
vCard option

### DISCUSSION THREAD

**Discussion by pxd303** [JulianPrieber/littlelink-custom#325] [Feature requests]
Id like to see an vcard option so users can add a "Add to Contacts" button!

**Comment by soluency** [comment]
We need this option!

**Reply by JulianPrieber** [reply]
Can you list some examples for this? I'm not sure how I would approach this :sweat_smile:.

**Reply by soluency** [reply]
https://vcardmaker.com/

**Reply by JulianPrieber** [reply]
How would you recommend implementing this into the site?

**Reply by soluency** [reply]
For "predefined site" you can have an option for vCard.  Then you can have fields for name, phone, organization, etc.  The default title can be something like "Download Contact".  The way I've been making vcards with your app is I created a directory with all the .vcf files and I use "Phone" as the "predefined site".  Then I just use that for the URL.  For instance: https://domain.com/vcf/file.vcf.

**Reply by Talk2Giuseppe** [reply]
> How would you recommend implementing this into the site?

Been awhile since you posted your question.  But I don't see a proper answer, so I'll add my thoughts.

The VCard link will behave the same way a QR Code VCard would behave.  In the sense of a QRCode, the reader after scanning the QR code prompts to add the contact to your address book.  Once added to contacts, you can further edit the data before saving.  A link would behave the same as the scan.  The OS will read the vcard tag and know what to do.  Probably best for the end user of LinkStack to put the data behind a short URL with the vcard stored on a server someplace.  Unless LinkStack allows for the vcard data file to remaining within the database of the host.

**Comment by ghost** [comment]
Would be great to allow vcards to have toggles to turn options off and on. An example would be turning off home address or work address. Would also love to see social profiles added or even payment methods etc.

**Reply by JulianPrieber** [reply]
As far as I can tell, I don't really have any control over that.
We are using a dependency to create the vcard. They have to implement that in their system first before we can use it here.
